tests/trustyai/drift/conftest.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.deployment import Deployment
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.namespace import Namespace
+from ocp_resources.secret import Secret
+from ocp_resources.serving_runtime import ServingRuntime
+from ocp_resources.trustyai_service import TrustyAIService
+
+from tests.trustyai.constants import MODELMESH_SERVING
+from tests.trustyai.drift.utils import wait_for_modelmesh_pods_registered_by_trustyai
+
+MLSERVER: str = "mlserver"
+MLSERVER_RUNTIME_NAME: str = f"{MLSERVER}-1.x"
+MLSERVER_QUAY_IMAGE: str = "quay.io/aaguirre/mlserver@sha256:8884d989b3063a47bf0e6c20c1c0ff253662121a977fe5b74b54e682839360d4"  # TODO: Move this image to a better place
```

Comment:
Is this a constant? Or does it change release by release?

Reply:
It's constant, and we already have it mirrored for disconnected envs

Reply:
Then it would make sense to have it in a constants file IMHO